static/js/message_list_view.js

```diff
@@ -697,7 +697,7 @@ MessageListView.prototype = {
         }
 
         this.clear_table();
-        this.render(this.list.all().slice(this._render_win_start,
+        this.render(this.list.all_messages().slice(this._render_win_start,
                                           this._render_win_end), 'bottom');
```

Comment:
Need to fix indentation on the second line of the block here (and in the one below)